src/filters.ts

```diff
@@ -81,6 +81,10 @@ export const TYPE_TO_FILTER_MAPPINGS = {
     '>=': ['greaterThanOrEqualTo', (value: Date) => value.toISOString()],
     default: ['equalTo', (value: Date) => value.toISOString()],
   },
+  // provided by postgraphile-plugin-fulltext-filter
+  SCALAR: {
+    default: ['matches', (value: string) => `${value}*`],
```

Comment:
i guess this needs to be a little more complex to allow users to use the full potential of fulltext queries.

eg `foo -bar` should not result in `foo -bar*`

we weiter leave the queries as is always,
or we need to scan it for the possible makers for an expert-query and only append * for non-expert queries

not sure about all possiblities, but this list looks like it is a good start
https://github.com/caub/pg-tsquery
maybe pg-tsquery (used by the plugin) can be used to see if the query is an expert query

Reply:
did not realize one can do things like that, but true. do you think the cms users know about that feature? maybe we should use the or query for each transmitted word.
`hallo welt` -> `hallo | welt`